rainbow-me/rainbowkit
# feat: `bybitWallet`

## Changes
- Added `bybitWallet`

## What to test
1. Install Bybit Wallet extension [here](https://chromewebstore.google.com/detail/bybit-wallet/pdliaogehgdbhbnmkklieghmmjkpigpa)
2. Install Bybit app [here](https://apps.apple.com/gb/app/bybit-buy-trade-crypto/id1488296980)
3. After doing step 1 and step 2, now test the extension and wallet app with WalletConnect and make sure the flow works as expected
